app/pings.yaml

```diff
@@ -18,3 +18,14 @@ activation:
     - https://github.com/mozilla-mobile/fenix/pull/1707#issuecomment-486972209
   notification_emails:
     - fenix-core@mozilla.com
+
+installation:
+  description: >
+    Intended for counting user installs.
+  include_client_id: false
+  bugs:
+    - 7295
```

Comment:
Please put the full link to the issue here

Reply:
will fix